.circleci/config.yml

```diff
@@ -166,6 +166,21 @@ jobs:
       #    is triggered automatically via GitHub webhook on https://readthedocs.org/projects/git-machete/builds/,
       # 3) every time a https://github.com/r-ryantm bot runs on NixOS/nixpkgs repository,
       #    Nix git-machete package is updated with the fresh version/package hash according to the data from PyPI.
```

Comment:
Also! to fix the failing CI, pls rebase onto the most recent develop, I've patched a few things